resources/migrations/001_update_migrations.yaml

```diff
@@ -9383,6 +9383,242 @@ databaseChangeLog:
                   name: notification_id
             indexName: idx_notification_subscription_notification_id
 
+  - changeSet:
+      id: v51.2024-09-05T08:00:04
+      author: qnkhuat
+      comment: Create the notification_handler table
+      preConditions:
+        - not:
+          - tableExists:
+              tableName: notification_handler
+      changes:
+        - createTable:
+            tableName: notification_handler
+            remarks: which channel to send the notification to
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              # TODO: can be removed once emails and slack configuration are fully migrated to be stored in channel table.
+              - column:
+                  name: channel_type
+                  type: varchar(64)
+                  remarks: the type of the channel
```

Comment:
Worth mentioning the concrete values here?

Reply:
added